src/library/formatCurrencyAmount.js

```
@@ -0,0 +1,20 @@
+/**
+ * Formats a given number as a USD currency string.
+ *
+ * @param {number|string} amount - The number to format. Can be a number or a numeric string.
+ * @returns {string} - The formatted currency string (e.g., "$1,234.56").
+ */
+const formatCurrencyAmount = (amount) => {
+  if (isNaN(amount) || amount === null || amount === undefined) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)